pkg/model/manifest.go

```diff
@@ -944,6 +944,12 @@ func (manifest *Manifest) ExpandEnvVars() error {
 		}
 	}
 	if manifest.Destroy != nil {
+		if manifest.Destroy.Image != "" {
+			manifest.Destroy.Image, err = envsubst.String(manifest.Destroy.Image)
+			if err != nil {
+				return errors.New("could not parse env vars for an image used for remote destroy")
```

Comment:
I think this code is shared for remote and local destroy, right?

Reply:
No, when an `image` filed is specified, the operation is only remote. So the check in line `947` ensures that this would affect only to remote operation